src/app/hierarchical-grid/hierarchical-grid-column-moving-styled/hierarchical-grid-moving-styled.component.html

```diff
@@ -83,25 +83,25 @@
     </igx-column>
 
     <igx-row-island [key]="'Orders'" [autoGenerate]="false">
```

Comment:
Set row island moving property to 'true', so the columns in the child grids to be movable as before